src/lib389/lib389/cli_conf/replication.py

```diff
@@ -633,7 +633,7 @@ def create_repl_manager(inst, basedn, log, args):
     if args.passwd is not None:
         repl_manager_password = args.passwd
     elif args.passwd_file is not None:
```

Comment:
Shouldn't it be args.bind_passwd_file ?

Reply:
IMHO @tbordaz is right: 
bind-password-file is used in all sub-parsers except in repl_add_manager_parser  
So the right way to solve the issue is probably to change the parser 
(but keeping the --password-file as alias to avoid compatibility issue)
i.e: repl_add_manager_parser.add_argument('--bind-password-file', '--passwd-file', help="File containing the password")